inputs.tf

```diff
@@ -108,7 +108,7 @@ variable stac_server_inputs {
   })
   default       = {
     app_name                                      = "stac_server"
-    version                                       = "v2.2.3"
+    version                                       = "v3.1.0"
```

Comment:
Might be out of scope for this PR, but is there anyway to remove the duplication of this version string here and in `default.tfvars` (and elsewhere)?